cvat/apps/quality_control/serializers.py

```diff
@@ -115,10 +116,25 @@ class Meta:
             """,
             "oks_sigma": """
                 Like IoU threshold, but for points.
-                The percent of the bbox area, used as the radius of the circle around the GT point,
+                The percent of the bbox side, used as the radius of the circle around the GT point,
```

Comment:
FWIW, I'm not sure "side" is appopriate here, seeing as a box has two different side lengths. Maybe something like "geometric mean of bbox sides" would be better? Up to you.

Reply:
Ok, sounds good.